### ISSUE TITLE
Possible bug relating to VS Code, Quick Fix Code Action, ESLint and Prettier

### ISSUE THREAD

**Issue by miller-productions** [ota-meshi/astro-eslint-parser#202]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I'm using [eslint-plugin-astro](https://github.com/ota-meshi/eslint-plugin-astro). (`*.astro` file linting does not work with the parser alone. You should also use eslint-plugin-astro with it.)
- [X] I'm sure the problem is a parser problem. (If you are not sure, search for the issue in [eslint-plugin-astro](https://github.com/ota-meshi/eslint-plugin-astro) repo and open the issue in [eslint-plugin-astro](https://github.com/ota-meshi/eslint-plugin-astro) repo if there is no solution.
- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.

### What version of ESLint are you using?

8.40.0

### What version of `eslint-plugin-astro` and ` astro-eslint-parser` are you using?

- astro-eslint-parser@0.14.0

- eslint-plugin-astro <-- I tried removing this to see if the bug triggered without it also. For context, see [previous issue I reported, and closed](https://github.com/ota-meshi/eslint-plugin-astro/issues/203)

### What did you do?

<details>
<summary>Configuration - .eslintrc.cjs</summary>

```
module.exports = {
  extends: ['plugin:prettier/recommended'],
  overrides: [
    {
      // Define the configuration for `.astro` file.
      files: ["*.astro"],
      // Allows Astro components to be parsed.
      parser: "astro-eslint-parser",
      rules: {},
    },
  ],
};
```
</details>

```astro
<!-- Paste your code here -->
```

### What did you expect to happen?

Configuring VSCode ESLint to use `plugin:prettier/recommended` would work in `.astro` files.

Steps to demonstrate:

1. Configure a basic astro project repo
2. Install VSCode extensions:
- [ESLint](https://github.com/Microsoft/vscode-eslint)
- [Astro](https://github.com/withastro/language-tools/tree/main/packages/vscode)
- [Prettier](https://github.com/prettier/prettier-vscode)
3. Install devDependecies:
```
    "astro-eslint-parser": "^0.14.0",
    "eslint": "^8.40.0",
    "eslint-config-prettier": "^8.8.0",
    "eslint-plugin-prettier": "^4.2.1",
    "prettier": "2.8.8"
```
4. In a `.astro` file, cause something that breaks a Prettier rule, VSCode should show a `Problem` (underlined in red)
5. Hover over the underlined problem.

Expected Result
- VSCode should pop up a window with explanation of the `eslint(prettier)` problem _plus_ a `Quick Fix` code action option that you could click on to auto fix the problem

### What actually happened?

1. Open the `Output` console in VSCode and switch to see ESLint output.
2. Observe that when you hover over the underlined problem, an error occurs and no `Quick Fix` code action is available.

Error message displayed is:

```
[Error - 8:19:40 PM] Request textDocument/codeAction failed.
  Message: Request textDocument/codeAction failed with message: Invalid regular expression: //** eslint-disable-next-line/: Nothing to repeat
  Code: -32603 
```

To see where the Quick Fix Code Action for ESLint(Prettier) works, see screenshot of .eslintrc file. In this one, I've used single quotes to trigger an eslint(prettier) error. If you hover over the red underlined error, all works correctly
<img width="455" alt="image" src="https://user-images.githubusercontent.com/15177547/236992222-6040fabd-0893-4d83-b208-b8a90e723c56.png">

To see where the Quick Fix Code Action for ESLint(Prettier) does not work, see screenshot of index.astro file. In this one, I've used single quotes to trigger an eslint(prettier) error, and also a missing semicolon. If you hover over the red underlined error in either of those two, the error occurs.

<img width="461" alt="image" src="https://user-images.githubusercontent.com/15177547/236992500-e7005000-78fa-4d7b-9373-eb0e347608e0.png">

### Link to Minimal Reproducible Example

https://github.com/miller-productions/astro-eslint-prettier-bug

**Comment by ota-meshi** [comment]
Thank you for the report.

I don't know the cause yet, but adding the following to `vscode/settings.json` seems to work fine:

```json
  "eslint.codeAction.disableRuleComment": {
    "enable": false,
  }
```

**Comment by ota-meshi** [comment]
It seems to be an issue with VSCode ESLint.
https://github.com/microsoft/vscode-eslint/issues/1610
But it's already fixed but not released yet.
I can't do anything with this repository so I'm closing the issue.

**Comment by miller-productions** [comment]
Understood. Thanks for the link and explanation @ota-meshi  🙂

Will keep an eye out for the related fix to be released in vscode-eslint.

**Comment by miller-productions** [comment]
FYI - they've incorporated the fix for this in `Version 2.4.1 - Pre-release`
